pkgs/development/libraries/audio/roc-toolkit/default.nix

```diff
@@ -49,6 +49,10 @@ stdenv.mkDerivation rec {
        else [ "--with-libraries=${openfec}/lib"
               "--with-openfec-includes=${openfec.dev}/include" ]);
 
+  prePatch = if stdenv.isAarch64 then "
```

Comment:
i would prefer a `lib.optional` instead of an `if`/`else`.

Reply:
Done.